test/stores/filters_store_test.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:economiza_sc/models/market.dart';
+import 'package:economiza_sc/modules/app_module.dart';
+import 'package:economiza_sc/stores/filters_store.dart';
+import 'package:flutter_modular/flutter_modular.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mobx/mobx.dart' as mobx;
+import 'package:mockito/mockito.dart';
+
+void main() {
+  final market = Market(id: 1, name: '', website: '');
+
+  setUp(() {
+    Modular.init(AppModule());
+  });
+
+  tearDown(() {
+    Modular.destroy();
+  });
+
+  group('FilterStore class', () {
+    group('updateSearch method', () {
+      test('The value of search is updated succesfully', () {
+        String newValue = 'newValue';
+        final filterStore = FiltersStore();
+        filterStore.search = mobx.Observable('');
+        when(filterStore.updateSearch(newValue));
+
+        expect(filterStore.search.value, newValue);
+      });
+    });
+    group('changeSelectedMarkets', () {
```

Comment:
Let's improve the description of this test by changing it too: when the action "changeSelectedMarkets" is called with a market flagging it as selected, the market will be added to the list of selected markets.

Reply:
Follow the same logic and improve the other tests too.

Reply:
Got it, I'll try to be more descriptive with tests, thanks for the feedback